woostarter/patterns/about-left-aligned-image-description-button.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Title: Image on the left, description on the right
+ * Slug: woostarter/left-aligned-image-description-button
+ * Categories: text, about, media
+ * Keywords: about, text
+ * Description: A section with an image on the left, description and button on the right.
+ *
+
+?>
+<!-- wp:group {"metadata":{"name":"Image on the left, description on the right"},"align":"full","style":{"spacing":{"margin":{"top":"0","bottom":"0"}}},"layout":{"type":"default"}} -->
+<div class="wp-block-group alignfull" style="margin-top:0;margin-bottom:0"><!-- wp:columns {"align":"wide","className":"is-style-default","style":{"spacing":{"blockGap":{"left":"0"}}}} -->
+<div class="wp-block-columns alignwide is-style-default"><!-- wp:column {"verticalAlignment":"center","width":"50%"} -->
+<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:50%"><!-- wp:cover {"url":"https://kiri.mystagingwebsite.com/wp-content/uploads/2025/02/pattern-ceramic-decoration.jpg","id":417,"dimRatio":0,"customOverlayColor":"#dedede","isUserOverlayColor":false,"isDark":false,"sizeSlug":"full","style":{"dimensions":{"aspectRatio":"4/3"}},"layout":{"type":"default"}} -->
```

Comment:
This `src` attribute needs to be updated to point to the themes assets directory (example: `<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/pattern-ceramic-decoration.jpg`) before merging.